.eslintrc.json

```diff
@@ -16,7 +16,12 @@
   ],
   "plugins": ["@typescript-eslint", "prettier", "react", "react-hooks"],
   "rules": {
-    "prettier/prettier": ["error"],
+    "prettier/prettier": [
+      "error",
+      {
+        "endOfLine": "auto"
```

Comment:
Is this fixing something specific?

Reply:
thanks for the comment,I think I was getting some prettier errors when committing a while ago without the end of line auto but it can definitely be removed.

Reply:
Let's remove this for now, we don't know how is gonna affect the rest of the files.

Reply:
got it! should be removed now